cfpb/cfgov-refresh
# Add notification molecule to doc detail page

## Changes

- Add notification molecule to document detail page type in Wagtail.

## Testing

1. Pull branch, run migrations and run server.
2. Edit `/policy-compliance/guidance/supervisory-guidance/bulletin-indirect-auto-lending-compliance/` to add a notification like the one that's there.
3. Preview the page to see the notification added.

## Screenshots

<img width="809" alt="Screen Shot 2019-09-25 at 6 04 06 PM" src="https://user-images.githubusercontent.com/704760/65643445-c6e2a300-dfbf-11e9-8e77-f47518dcf821.png">

![Uploading Screen Shot 2019-09-25 at 6.06.10 PM.png…]()
